dj_anonymizer/anonymizer.py

```diff
@@ -23,55 +17,44 @@ def __init__(self, soft_mode=True):
         models_set = set()
         for app in apps.get_app_configs():
             if app.name not in settings.ANONYMIZER_SKIP_APPS:
-                models_set.update(model.__module__ + '.' + model.__name__ for model in app.get_models())
+                models_set.update(
+                    model.__module__ + '.' + model.__name__ for model in app.get_models()
+                )
+
+        all_models = set(self.skip_models + self.anonym_models.keys() + self.clean_models.keys())
 
-        if not soft_mode and not models_set.issubset(set(self.skip_models + self.anonym_models.keys() + self.clean_models.keys())):
-            print 'You did not set models to any list:'
-            print list(models_set.difference(set(self.skip_models + self.anonym_models.keys() + self.clean_models.keys())))
-            raise LookupError
+        if not soft_mode and not models_set.issubset(all_models):
+            raise LookupError('You did not set those models to any list: {}'.format(
+                              list(models_set.difference(all_models))))
 
     def anonymize(self):
         print 'Updating started'
         for anonym_cls in self.anonym_models.values():
-            if not anonym_cls._get_fields_names():
+            if not anonym_cls.get_fields_names():
                 continue
 
-            queryset = anonym_cls.Meta.queryset.only(*anonym_cls._get_fields_names())
+            queryset = anonym_cls.Meta.queryset.only(*anonym_cls.get_fields_names())
             print '\nGenerating fake values for model "{}"'.format(queryset.model.__name__)
+
             i = 0
             total = queryset.count()
             for j in range(0, total, ANONYMIZER_SELECT_BATCH_SIZE) + [None]:
                 sub_set = queryset.order_by('pk')[i:j]
                 for model in sub_set:
                     i += 1
-                    stdout.write('\rProgress: {}%'.format(int(math.floor((i / float(total) * 100)))))
-                    stdout.flush()
 
-                    for name in anonym_cls._get_fields_names():
+                    for name in anonym_cls.get_fields_names():
                         if getattr(model, name) or anonym_cls.Meta.fill_empty:
                             setattr(model, name, next(getattr(anonym_cls, name)))
 
-                bulk_update(sub_set, batch_size=ANONYMIZER_UPDATE_BATCH_SIZE, update_fields=anonym_cls._get_fields_names())
+                bulk_update(sub_set,
+                            batch_size=ANONYMIZER_UPDATE_BATCH_SIZE,
+                            update_fields=anonym_cls.get_fields_names())
         print '\n\nUpdating finished'
 
     def clean(self):
         print '\nCleaning started\n'
-
-        warning_models = []
-        for model_name, queryset in self.clean_models.items():
```

Comment:
why have you deleted it?

Reply:
@cubicova17 we 2 versions of dj_anonymizer, the first is in apollo repo, which we use, the second is in dj_anonymizer repo, which is abandoned and unused. 
That particular deletition was made in apollo repo by someone for some reason, I just copied working version (from apollo repo) and refactored it.
If you want, I can bring it back, but we are living without it now.